### ISSUE TITLE
Select debug program and switch between running debug programs

### ISSUE THREAD

**Issue by Trenrod** [Microsoft/vscode#21127]
Its a feature request. Debugging multiple programs without using a mouse.

So far you can only start one preselected debug program with F5.
Its not possible without using a mouse to select a specific debug program, even to focus it with cursor or tab navigation.

![image](https://cloud.githubusercontent.com/assets/5102094/23212747/74bb569e-f908-11e6-9356-d1ad5a260f7d.png)

Same if you run multiple programs in debug you cannot switch between them to e.g. restart

![image](https://cloud.githubusercontent.com/assets/5102094/23212786/82003f72-f908-11e6-9de6-b2444ef6438a.png)

Thank you.

**Comment by michelkaporin** [comment]
Thank you for raising it up. Luckily this was already requested in #16613 and implemented in #20979. It is currently available in Insiders build and will go into the next release.

Closing as duplicate.

**Comment by Trenrod** [comment]
Just saw that 1.10.1 added the feature to select a launch config.

But I cant find a solution for the second request. Which is to select between running programs.

Thank you.

**Comment by isidorn** [comment]
Yes the second request is not captured by this. Let's reopen this issue

**Comment by isidorn** [comment]
This is currently possible by 
F1 > focus call stack > press 'enter' on the session you want focused

As an alternatvie I was looking into adding an action which would open quick open so the uer could choose which session to focus. However this seems to take the same amount of time as the already available solution. Thus I choose not to add it to remove duplication
